dataikuapi/dss/apideployer.py

```diff
@@ -334,15 +334,29 @@ def start_update(self):
 
         return DSSFuture(self.client, future_response.get('jobId', None), future_response)
 
-    def delete(self):
+    def delete(self, force=False):
         """
-        Deletes this deployment
+        Deletes this deployment.
+
+        :param boolean force: If True, automatically disables deployment before deletion
 
         You may only delete a deployment if it is disabled and has been updated after disabling it.
```

Comment:
I would put this sentence above on line 339 (right after `Deletes this deployment.`)